--- libcdaudio-0.99.4/libcdaudio.m4.orig	1999-12-26 19:54:01.000000000 +0100
+++ libcdaudio-0.99.4/libcdaudio.m4	2004-02-12 23:21:25.403804264 +0100
@@ -7,7 +7,7 @@
 dnl Test for libcdaudio, and define LIBCDAUDIO_CFLAGS, LIBCDAUDIO_LIBS and
 dnl LIBCDAUDIO_LDADD
 dnl
-AC_DEFUN(AM_PATH_LIBCDAUDIO,
+AC_DEFUN([AM_PATH_LIBCDAUDIO],
 [dnl 
 dnl Get the cflags and libraries from the libcdaudio-config script
 dnl
